### PR TITLE
feat(): Add SRTM Downloading for offline usage

### PR DESCRIPTION
**Pull request by julled**
We download a specific geojson area for later offline usage.

This will be needed for autolanding purposes.

### CODE REVIEW

File: stage3/searchwing-antennabox/rootfs/home/searchwing/srtm_downloader.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import json
+import math
+from tqdm import tqdm
```
Comment (booo) [inline stage3/searchwing-antennabox/rootfs/home/searchwing/srtm_downloader.py:3]: Do we actually need the tqdm progressbar? If we need it I guess we have to install it somewhere.

File: stage3/searchwing-antennabox/01-run.sh [hunk 2]
```diff
@@ -48,3 +49,9 @@ on_chroot << EOF
   pip install git+https://github.com/booo/mavproxy@movinghome-gpsd-searchwing
   pip install gpsdclient
 EOF
+
+install -m 644 -o ${FIRST_USER_NAME} -g ${FIRST_USER_NAME} "rootfs/home/searchwing/eberswalde.geojson" "${ROOTFS_DIR}/home/${FIRST_USER_NAME}/"
```
Comment (booo) [inline stage3/searchwing-antennabox/01-run.sh]: Could you add data for the other airfields too?

Comment (julled) [reply stage3/searchwing-antennabox/01-run.sh]: which ones

Comment (julled) [reply stage3/searchwing-antennabox/01-run.sh]: but i will wait until it works

File: stage3/searchwing-antennabox/rootfs/home/searchwing/srtm_downloader.py (hunk 1 above)
Comment (julled) [reply stage3/searchwing-antennabox/rootfs/home/searchwing/srtm_downloader.py:3]: fixed

**Comment by julled** [comment]
Integration is done, would you like to test if in the image the SRTM data is located in `/home/searchwing/.tilecache`?

File: stage3/searchwing-antennabox/01-run.sh (hunk 2 above)
Comment (julled) [reply stage3/searchwing-antennabox/01-run.sh]: download might worked, which other airfields do yo uwant?

**Comment by booo** [comment]
> Integration is done, would you like to test if in the image the SRTM data is located in `/home/searchwing/.tilecache`?

Yes, let us test one of the images from the CI and merge this then. I can do this when I am back from my trip.

**Comment by julled** [comment]
lets merge this?

**Comment by booo** [comment]
Happened already in 4158b85f280a1bc770fc7e342b15e2bbae57bbbd

**Pull request closed without merging**